ares/n64/pi/bus.hpp

```diff
@@ -6,55 +6,138 @@ inline auto PI::readWord(u32 address, Thread& thread) -> u32 {
     thread.step(writeForceFinish() * 2);
     return io.busLatch;
   }
-  thread.step(250 * 2);
-  io.busLatch = busRead<Word>(address);
+  if (system._BB()) {
```

Comment:
Also discussed on Discord: the PI changes should go into a separate directory/module because they are very pervasive within these files.